anta/decorators.py

```diff
@@ -56,27 +78,46 @@ async def wrapper(*args: Any, **kwargs: Any) -> TestResult:
 
 
 def check_bgp_family_enable(family: str) -> Callable[[F], F]:
-    """Return a decorator factory to skip a test if BGP is enabled.
+    """
+    Return a decorator to conditionally skip or fail a test based on BGP address family availability.
```

Comment:
in this decorator lets mark as part of this PR that we are planning on deprecating it